src/frontend/libslirp/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+cmake_minimum_required(VERSION 3.16)
+
+project(libslirp VERSION 4.8.0 LANGUAGES C)
+
+set(SLIRP_MAJOR_VERSION "${libslirp_VERSION_MAJOR}")
+set(SLIRP_MINOR_VERSION "${libslirp_VERSION_MINOR}")
+set(SLIRP_MICRO_VERSION "${libslirp_VERSION_PATCH}")
+set(SLIRP_VERSION_STRING "\"${libslirp_VERSION}\"")
+
+set(SOURCES
```

Comment:
My stub is smaller because I don't compile all of slirp; just the parts that melonDS uses. These are the files I build:

```cmake
target_sources(slirp PRIVATE
    "${libslirp_SOURCE_DIR}/src/arp_table.c"
    "${libslirp_SOURCE_DIR}/src/bootp.c"
    "${libslirp_SOURCE_DIR}/src/cksum.c"
    "${libslirp_SOURCE_DIR}/src/dhcpv6.c"
    "${libslirp_SOURCE_DIR}/src/dnssearch.c"
    "${libslirp_SOURCE_DIR}/src/if.c"
    "${libslirp_SOURCE_DIR}/src/ip_icmp.c"
    "${libslirp_SOURCE_DIR}/src/ip_input.c"
    "${libslirp_SOURCE_DIR}/src/ip_output.c"
    "${libslirp_SOURCE_DIR}/src/ip6_icmp.c"
    "${libslirp_SOURCE_DIR}/src/ip6_input.c"
    "${libslirp_SOURCE_DIR}/src/ip6_output.c"
    "${libslirp_SOURCE_DIR}/src/mbuf.c"
    "${libslirp_SOURCE_DIR}/src/ndp_table.c"
    "${libslirp_SOURCE_DIR}/src/ncsi.c"
    "${libslirp_SOURCE_DIR}/src/sbuf.c"
    "${libslirp_SOURCE_DIR}/src/slirp.c"
    "${libslirp_SOURCE_DIR}/src/socket.c"
    "${libslirp_SOURCE_DIR}/src/tcp_input.c"
    "${libslirp_SOURCE_DIR}/src/tcp_output.c"
    "${libslirp_SOURCE_DIR}/src/tcp_subr.c"
    "${libslirp_SOURCE_DIR}/src/tcp_timer.c"
    "${libslirp_SOURCE_DIR}/src/tftp.c"
    "${libslirp_SOURCE_DIR}/src/udp.c"
    "${libslirp_SOURCE_DIR}/src/udp6.c"
    "${libslirp_SOURCE_DIR}/src/util.c"
)
```